docs/guides/alias.rst

```diff
@@ -49,6 +49,8 @@ When set to 'yes', PowerDNS will look up the A and AAAA records of the name in t
 This is useful when your secondary servers do not understand ALIAS, or should not look up the addresses themselves.
 Note that secondaries will not automatically follow changes in those A/AAAA records unless you AXFR regularly.
 
+If secondaries are operated with nameserver software which is not capable of synthesising ALIAS records the zone will probably go out of sync. Because updated ALIAS target records will not result in an increment of the Serial, which means that Secondaries won't pull the Zone.
```

Comment:
```suggestion
If secondaries are operated with name server software which is not capable of synthesizing ALIAS records the zone will probably go out of sync. Because updated ALIAS target records will not result in an increment of the Serial, which means that Secondaries won't pull the Zone.
```